srcpkgs/spago/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+makedepends="
+	ncurses-libtinfo-libs
+	ncurses-libtinfo-devel
+"
```

Comment:
unnessecary once fixed

Reply:
When building from binary, these are required so `stack` knows which GHC to build. It uses `libtinfo.so.6`